Source/Schema.NET/Schema.NET.csproj

```diff
@@ -45,6 +45,10 @@
     <PackageReference Include="Microsoft.VisualStudio.Threading.Analyzers" Version="16.4.43" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.1' Or '$(TargetFramework)' == 'net461'">
+    <PackageReference Include="System.ValueTuple" Version="4.5.0" />
```

Comment:
This dependency is required for the constructor delegate lookup in `FastActivator` though is only needed in two targets.